Makefile

```diff
@@ -44,6 +44,6 @@ download-snap:
 	wget -q https://github.com/canonical/openstack-exporter-operator/releases/download/rev2/golang-openstack-exporter_amd64.snap -O ./golang-openstack-exporter_amd64.snap
 
 integration: build download-snap
-	CHARM_LOCATION=${CHARM_LOCATION} CHARM_SNAP_LOCATION=${CHARM_SNAP_LOCATION} tox -e integration
+	MODEL_SETTINGS="update-status-hook-interval=30s" CHARM_LOCATION=${CHARM_LOCATION} CHARM_SNAP_LOCATION=${CHARM_SNAP_LOCATION} tox -e integration
```

Comment:
**nitpick:** 

Can we make it part of https://github.com/canonical/bootstack-actions/blob/7bf3d7fb3a3d43302e35a46453c7a513df196571/.github/workflows/func.yaml#L83 ?

Reply:
Yes / no: yes, I think it's general safe to set a low update status hook interval in the CI; no, I don't have information to update `zaza-yaml` CI env

Reply:
Not done, need the old `zaza-yaml` which I don't have

Reply:
It's a base64 encoded string you can found here: https://github.com/canonical/openstack-exporter-operator/blob/882043f084a4b9164e944a964e366e12cd551e22/.github/workflows/_quality_check.yaml#L76C19-L76C175